blocks/reader-revenue-manager/block-editor-plugin/SettingPanel.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Reader Revenue Manager Block Editor SettingPanel component.
+ *
+ * Site Kit by Google, Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+// eslint-disable-next-line import/no-unresolved
```

Comment:
Hey @ankitrox, one more thing - we no longer to disable `import/no-unresolved` on a per-line basis, as this is handled globally via ESLint config:

https://github.com/google/site-kit-wp/blob/aadae534f6f152ca837ae4e643d3bdfb73e7b5eb/.eslintrc.json#L17-L24